bindings/python/pyiree/compiler2/core.py

```diff
@@ -15,6 +15,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# TODO(#4131) python>=3.7: Use postponed type annotations.
```

Comment:
Is this comment applicable everywhere? It doesn't necessarily seem super useful to have the TODO listed everywhere as opposed to just listing it in the issue.

Reply:
It only affects files with cases like the following

```python
class ReferenceSelf

  @staticmethod
  def clone(self) -> "ReferenceSelf"
    pass
```

and `BackendInfo` (which can't be defined before it's used in `*CompiledModule` because it stores the `*CompiledModule` classes on its `cls`.